modules/ip6/control/address.c

```diff
@@ -135,9 +136,10 @@ static int ip6_mcast_addr_del(struct iface *iface, const struct rte_ipv6_addr *i
 }
 
 static int
-iface6_addr_add(const struct iface *iface, const struct rte_ipv6_addr *ip, uint8_t prefixlen) {
+iface6_addr_add(const struct iface *iface, const struct rte_ipv6_addr *ipv6, uint8_t prefixlen) {
 	struct hoplist *addrs;
 	unsigned addr_index;
+	struct rte_ipv6_addr *ip = (struct rte_ipv6_addr *)ipv6;
```

Comment:
if we need to change the signature of the function we can discuss it, no worries. But please no hacks like this.